### DISCUSSION TITLE
pregunta  sobre el ejercicio 9 del Tp2 (y uso de funciones)

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#59] [Preguntas y respuestas] [answered]
Hola! estaba leyendo la consigna del ejercicio 9 y no me había quedado claro si hay que sumar los números o si hay que sumar los _dígitos_ de los números (como el ejercicio 2 del tp2). y en el caso que sea así, esta permitido llamar otras funciones dentro de otras funciones que no sean 'main' para "segmentalizar" un poco mas ordenado todo?

**Comment by martinvilu** [comment]
Si podés copiar la consigna, te puedo contestar ahora, que estoy fuera de la computadora.

**Reply by rinrot** [reply]
Cálculo de Suma de Dígitos en Serie de Fibonacci

Escribe una función que calcule la suma de los dígitos de los términos en una serie de Fibonacci hasta el N-ésimo término.

**Reply by martinvilu** [reply, marked as answer]
> Hola! estaba leyendo la consigna del ejercicio 9 y no me había quedado claro si hay que sumar los números o si hay que sumar los dígitos de los números (como el ejercicio 2 del tp2). 

Los dígitos de cada uno de los números de la serie de Fibonacci hasta el n-esimo.

> y en el caso que sea así, esta permitido llamar otras funciones dentro de otras funciones que no sean 'main' para "segmentalizar" un poco mas ordenado todo?

Totalmente, por ahora, es copiar y pegar las funciones de otros ejercicios, pero eventualmente será algo parecido al `import` de Python.

**Reply by martinvilu** [reply]
¿Esta cerrado esto?

**Reply by rinrot** [reply]
si, puse el emoji pero no el tick, mala mia

**Reply by martinvilu** [reply]
Joyia
